.gitignore

```diff
@@ -35,6 +35,9 @@
 # Ignore downloaded temporary files, such as the trusted root CA certificates.
 /downloads/
 
+# Ignore generated binary example files
+**/examples/**/bin
```

Comment:
Is this directory used to store Wasm files and serialized application configs?

Reply:
Yes

Reply:
I think we should just pick a directory name (`bin` is fine, `out` was also suggested) and use it to generate everything everywhere (not just examples) (including binaries, wasm modules, additional stuff), rather than trying to get things out from `target/wasm32-unknown-unknown` and similar.

For `cargo build`, there is an `--out-dir` flag that should do that already.

Reply:
Just pointing out that I believe --out-dir is still experimental